pkg/server/api_v2_databases_metadata_test.go

```diff
@@ -894,11 +898,20 @@ func makeApiRequest[T any](
 	return mdResp
 }
 
-func assertJobTriggered(t *testing.T, client http.Client, url string, c chan interface{}) {
-	resp := makeApiRequest[tmJobTriggeredResponse](t, client, url, http.MethodPost)
-	require.Contains(t, resp.Message, "Job triggered successfully")
-	require.True(t, resp.JobTriggered)
-	<-c
+func assertJobTriggered(
```

Comment:
nit: should this function name be different? I would prefer `triggerAndWaitForJobToComplete`

Reply:
Done